Sources/MapboxDirections/MapMatching/MapMatchingResponse.swift

```diff
@@ -1,49 +1,55 @@
 import Foundation
 
 public struct MapMatchingResponse {
-    public var code: String
-    public var routes : [Route]?
-    public var waypoints: [Waypoint]
+    public let httpResponse: HTTPURLResponse?
+    
+    public var matches : [Match]?
+    public var tracepoints: [Tracepoint?]?
+    
+    public let options: MatchOptions
+    public let credentials: DirectionsCredentials
```

Comment:
I am not sure to understand why we need `credentials` in both `MapMatchingResponse` and `RouteResponse`.
Is it really important to know in a response which host or access token it used? Is it the case that there might be multiple hosts and/or tokens used within one app, so user needs to distinguish them?

Reply:
Yes, generally the credentials are important for purposes of re-routing, which can happen anytime during a navigation session.